guides/release/templates/input-helpers.md

```diff
@@ -48,7 +48,7 @@ current context.
 To dispatch an action on specific events such as `key-press`, use the following
 
 ```handlebars
-{{input value=this.firstName key-press=(action "updateFirstName")}}
+<Input @value={{this.firstName}} @key-press={{action "updateFirstName"}} />
 ```
 
 The following event types are supported (dasherized format):
```

Comment:
I think the API docs PR refers to these as "custom events".. double check to make sure and try to align the language